sdk/js/src/Request.ts

```diff
@@ -112,10 +112,14 @@ export const getConfigJson = async (
                 `response message: ${ex.message}` +
                 (errorString ? `, response data: ${errorString}` : ''),
         )
-        throw new Error(
-            `Failed to download DevCycle config.` +
-                (errorString ? ` Error details: ${errorString}` : ''),
-        )
+        if(ex?.message !== 'Network Error') {
+            throw new Error(
+                `Failed to download DevCycle config.` +
+                    (errorString ? ` Error details: ${errorString}` : ''),
+            ) 
+        } 
+        return { } as BucketedUserConfig
```

Comment:
this doesn't make sense. The client already catches errors thrown by requests and stays in a "no config" state

Reply:
I thought it would make sense to add an error handling on this call as well for the Network Error types, which seems to be crashing the app when initialization is failed. I also do not see proper error handling on the client, I am assuming that the Error will crash the SDK currently